index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Hello Studio Website.">
+    <title>Hello Studio</title>
+  </head>
+  <body>
+    <header>
+      <h1>
+        <span>the time is</span>
+        <span>now</span>
+      </h1>
+      <nav>
+        <ul>
+          <li><a href="/">intro</a></li>
+          <li><a href="/">gallery</a></li>
+          <li><a href="/">services</a></li>
+          <li><a href="/">contact</a></li>
+        </ul>
+      </nav>
+      <section>
+        <ul>
+          <li><a href="https://pinterest.com"><img src="assets/pinterest.svg" aria-hidden="true"></a></li>
+          <li><a href="https://facebook.com"><img src="assets/facebook.svg" aria-hidden="true"></a></li>
+          <li><a href="https://twitter.com"><img src="assets/twitter.svg" aria-hidden="true"></a></li>
+        </ul>
+      </section>
+    </header>
+    <section>
+      <h2>we're hello studio.</h2>
+      <p>
+        Lomo aesthetic lo-fi etsy sustainable cray. Typewriter knausgaard narwhal, vape af tofu occupy pour-over
+        truffaut chillwave cred retro normcore vice brooklyn. Salvia occupy pork belly literally. 3 wolf moon echo park
+        literally brooklyn gentrify, roof party seitan direct trade pickled cronut flexitarian. Try-hard knausgaard
+        biodiesel keffiyeh, 90's yr flexitarian man braid pickled messenger bag PBR&B asymmetrical. Listicle vegan XOXO
+        ugh direct trade, single-origin coffee master cleanse scenester. Lyft neutra godard normcore, swag tacos trust
+        und meh pitchfork dreamcatcher pork belly.
+      </p>
+      <p>
+        Offal bitters vaporware yr small batch lomo. Narwhal put a bird on it next level, sriracha lyft fanny pack
+        health goth hashtag bespoke church-key austin photo booth. Keytar activated charcoal pour-over, brooklyn kogi
+        taxidermy vinyl fingerstache. Mumblecore pug craft beer, cliche typewriter leggings fixie pitchfork irony swag.
+        Poke pabst PBR&B, paleo distillery neutra schlitz. Pop-up mustache 8-bit authentic fashion axe, neutra marfa
+        blue bottle kickstarter health goth. Organic health goth lyft, gochujang lomo small batch fingerstache marfa hot
+        chicken.
+      </p>
+      <img src="assets/about-img.jpg" alt="a woman walking through a dessert">
+    </section>
+    <section>
+      <h2>gallery</h2>
+      <p>
+        Hoodie leggings offal, heirloom lo-fi pug freegan put a bird on it hella crucifix authentic beard biodiesel
+        letterpress salvia. Man bun direct trade PBR&B small batch biodiesel meditation, roof party letterpress beard
+        tumblr.
+      </p>
+    </section>
+    <section>
+      <h3>
```

Comment:
Not sure if that is a `h3`. Probably if it was a webiste designed not as a training mockup we would have a `<h2>Opening hours</h2>` and then it would be just a `<p>` with `<span>`s. If you wanted you could add this `h2` and we would then hide it with CSS. The webiste would make sense from the perspective of someone using a screen reader then.

Reply:
I wasn't sure how to do it, so thanks for clarifying it :)